### PR TITLE
Feature rdp

### PR DESCRIPTION
**Pull request by 195858**

### CODE REVIEW

File: GuideCreateModule.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Guide to Creating a Module
```
Comment (195858) [inline GuideCreateModule.md:1]: This file is only a stub. When do we plan to add content?

Comment (ricardojosegomezulmke) [reply GuideCreateModule.md:1]: I will do it later today, oversight.

File: lib/ansible/module_utils/network/solace/solace_utils.py [hunk 2]
```diff
@@ -1,18 +1,21 @@
 #!/usr/bin/env python
 
 # Copyright (c) 2019, Mark Street <mkst@protonmail.com>
+# Copyright (c) 2020, Solace Corporation, Ricardo Gomez-Ulmke <ricardo.gomez-ulmke@solace.com>
```
Comment (195858) [inline lib/ansible/module_utils/network/solace/solace_utils.py:4]: Do we need to keep the original copyright under MIT rules?

**Comment by ricardojosegomezulmke** [comment]
Replying from e-mail….


Yes, I believe we keep the original. I think we can add ours …


--
Ricardo Gomez-Ulmke
VP IoT
+44(0)7555 724 630
Imagine Event Mesh <https://www.youtube.com/watch?v=qv5lwINR0Ew&feature=youtu.be>  IoT Solutions Program<http://solace.com/partners/iot>

File: lib/ansible/module_utils/network/solace/solace_utils.py (hunk 2 above)
Comment (ricardojosegomezulmke) [reply lib/ansible/module_utils/network/solace/solace_utils.py:4]: ok, reply from e-mail doesn't work. 
Yes, I believe we keep the original. I think we can add ours …

File: lib/ansible/modules/network/solace/solace_bridge_remote_vpn.py [hunk 3]
```diff
@@ -8,24 +8,33 @@
...
-
-    def get_func(self, solace_config, vpn, virtual_router, bridge_name, remote_vpn_location):
-        """Pull configuration for all Bridges associated with a given VPN"""
+    def get_func(self, solace_config, vpn, virtual_router, bridge_name, remote_vpn_location, lookup_item_value):
+        # GET /msgVpns/{msgVpnName}/bridges/{bridgeName},{bridgeVirtualRouter}/remoteMsgVpns/{remoteMsgVpnName},{remoteMsgVpnLocation},{remoteMsgVpnInterface}
+        # here: omitting {remoteMsgVpnInterface}
         bridge_uri = ','.join([bridge_name, virtual_router])
-        path_array = [su.SEMP_V2_CONFIG, su.MSG_VPNS, vpn, su.BRIDGES, bridge_uri, su.BRIDGES_REMOTE_MSG_VPNS]
-        return su.get_configuration(solace_config, path_array, 'remoteMsgVpnName')
+        remote_vpn_uri = ','.join([lookup_item_value, remote_vpn_location])
```
[the remark this answers is not in the document]
Comment (195858) [reply lib/ansible/modules/network/solace/solace_bridge_remote_vpn.py]: Needed to add in the last separator

**Pull request merged**